content/blog/api-console-for-data-services-cloud-console.md

```diff
@@ -0,0 +1,200 @@
+
+The API Gateway menu is available inside the Manage menu. From HPE Greenlake Console click on Menu to access this Manage Menu.
+
+![Access Menu ](/img/accesing-manage-menu-from-console.png "Menu in Cloud Console")
+
+![CCS Menu](/img/accesing-api-gateway-from-manage-menu.png "GreenLake Common Cloud Menu")
+
+The API Gateway menu provides the following operations:
+
+1. Create and manage API client credential association in order to obtain:
+
+   1. Instance ID of the DSCC at the particular region (Hexadecimals).
+   2. Client ID (Hexadecimals).
+   3. Client Secret (Hexadecimals).
+   4. URL to the HPE GreenLake Console API end-point (string).
+   5. URL to the HPE DSCC API end-point (string).
+2. Generate Access Token, change Client Secret, and delete Access Token.
+
+![API Gateway](/img/api-gateway-block.png "DSCC API Gateway")
+
+### Manage API Client Applications
+
+Each instance of the API client credential represents the authorization relationship between the client application and the DSCC REST API end point. Click on the Create Credentials button to generate a client credential. Afterward the user can obtain the client secret and client secret, and used them to generate the access token.
+
+![](/img/create-credentials-button.png)
+
+Inside the Create Credentials menu, click on the V button to show the pull down list and use the mouse to click on the desired application. For the purpose of using the DSCC REST API, please select Data Services Cloud Console instance in the correct region where the array has been deployed. The list shows all instances with the region where the applications are deployed.
+
+![select the application](/img/select-the-desired-application.png "Choose application (DSCC)")
+
+After selecting the correct application, enter the Credential Name (Please see HPE GreenLake Cloud Console User Guide for supported characters).  Click the Create Credentials button to proceed with Client Credentials creation.
+
+![Create Credential no input yet](/img/create-credentials-menu.png "Generate Client Credentials 1st time")
+
+Once the Create Credential button is clicked, the following information about the OAuth (Open Authorization) will be shown. These are the Client ID and Client Secret information menu that is going to be shown only once. Please copy both Client ID and Client Secret to a safely recorded location. In the case that the user missed to copy these information, then the user can only regenerate the Client Secret at later time using the Client-Secret Reset menu.
```

Comment:
Change "menu" to "menus"
Change "is" to "are"
(So the sentence should read: "These are the Client ID and Client Secret information menus that are going to be shown only once.")
Change the last sentence to read:" In case the user missed copying this information, then the user can only regenerate the Client Secret at a later time using the Client Secret Reset menu." (first part of the sentence changed and the word "a" added prior to "later")